users/user-creation.md

```diff
@@ -0,0 +1,42 @@
+## User Creation
+
+Aside from [automatic account creation](/jetpack/automatic-account-creation-connection.md) there is an endpoint available for creating a WordPress.com user. This endpoint assumes the user creation is part of a product provisioning flow.
+
```

Comment:
It might be worth mentioning WooCommerce provisioning here. Perhaps something like:

> This endpoint assumes the user creation is part of a product provisioning flow, such as provisioning a WooCommerce package to someone that does not yet have a WordPress.com account.